services/web/client/source/class/qxapp/Application.js

```diff
@@ -73,12 +73,11 @@ qx.Class.define("qxapp.Application", {
     },
 
     __initRouting: function() {
-      // Route: /{id}
-      // let result = /study\/(.*)/.exec(window.location.pathname); FIXME: PC->IP
+      // Route: /#/study/{id}
       let result = /#\/study\/(.*)/.exec(window.location.hash);
-      if (result && result[result.index].length) {
+      if (result && result[1].length) {
```

Comment:
results.length > 0

Reply:
I thought it was wals the case? See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/exec#Description

Reply:
![image](https://user-images.githubusercontent.com/32402063/57471395-58f54380-728b-11e9-87af-521c34e088de.png)